composer.json

```
@@ -4,7 +4,7 @@
     "description": "Flow package that integrates MailChimp into your Neos site / Flow Application",
     "license": "GPL-3.0+",
     "require": {
-        "neos/neos": "^5.0"
+        "neos/neos": "^5.0 || ^7.0"
```

@jobee Is this really compatible with Neos 5 and Neos 7? I guess this will not work for Neos 5 anymore as the implementation doesn't meet the "old" `Neos\Flow\Persistence\QueryInterface` anymore.

So, shouldn't it be :
```suggestion
        "neos/neos": "^7.0"
```

---

@dlubitz are you sure this is not b/c?
Implementations can add _additional_ type hints that are not in the interface, see https://3v4l.org/Mth2X